contracts/mocks/unreachable/ERC721ConsecutiveEnumerableMock.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I would move this back to something like `contracts/mocks/ERC721ConsecutiveEnumerableMock.unreachable.sol` so that all ERC721 mocks are together